src/Asset/EntrypointsJsonLookup.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * Copyright (c) 2018 Heimrich & Hannot GmbH
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace HeimrichHannot\EncoreBundle\Asset;
+
+
+use Symfony\Component\DependencyInjection\Container;
+
+class EntrypointsJsonLookup
+{
+
+    public function mergeEntries(array $entrypointsJsons, array $entries, string $babelPolyfillEntryName = null) : array
+    {
+        foreach ($entrypointsJsons as $entrypointsJson) {
+            $entrypoints = $this->parseEntrypoints($entrypointsJson);
+
+            $entriesMap = [];
+            foreach ($entries as $entry) {
+                if (!isset($entry['name'])) {
+                    continue;
+                }
+
+                $entriesMap[$entry['name']] = true;
+            }
+
+            foreach ($entrypoints as $name=>$entrypoint) {
+                // Ignore the babel-polyfill entry
+                if ($name == $babelPolyfillEntryName) continue;
```

Comment:
I think this should only be skipped, if addBabelPolyfill is checked. In a feature major release I propose to remove the add babel-polifill option, as it can now be added this way.

Reply:
I wil change this to $babelPolyfillEntryName beeing null if addBabelPolyfill is not checked. But you are right we could probably remove that if this is merged.